api/facadeversions.go

```diff
@@ -106,7 +106,7 @@ var facadeVersions = map[string]int{
 	"Subnets":                      4,
 	"Undertaker":                   1,
 	"UnitAssigner":                 1,
-	"Uniter":                       15,
+	"Uniter":                       16,
```

Comment:
I bumped it just to be on the safe side because I wasn't sure about pylibjuju (which frustratingly includes the schema for the agent API calls in addition to the client ones). Based on the feedback from @hmlanigan and the above comment I will revert this change and revise the PR later today.